gammapy/data/data_store.py

```diff
@@ -51,37 +51,41 @@ def __str__(self):
         return self.info(show=False)
 
     @classmethod
-    def from_files(cls, base_dir, hdu_table_filename=None, obs_table_filename=None):
-        """Construct from HDU and observation index table files."""
-        if hdu_table_filename:
-            log.debug("Reading {}".format(hdu_table_filename))
-            hdu_table = HDUIndexTable.read(str(hdu_table_filename), format="fits")
+    def from_dir(cls, base_dir, hdu_table_filename=None, obs_table_filename=None):
+        """Create from a directory.
+
+        If no filenames are specified, files with default filenames
```

Comment:
Could you a `Parameters` section here instead, that describes the method arguments, like we do for every method in Gammapy? You will find a lot of examples how to do it in the same source file.